### ISSUE TITLE
BUG: Too many arguments error when using -s flag

### ISSUE THREAD

**Issue by mhdcodes** [jasonmccreary/git-trim#7]
## Description

Running the command with stale flag results in an error `git trim -s --dry-run`

```
~/.oh-my-zsh/custom/plugins/git-trim/git-trim: line 230: [: too many arguments
~/.oh-my-zsh/custom/plugins/git-trim/git-trim: line 230: [: too many arguments
~/.oh-my-zsh/custom/plugins/git-trim/git-trim: line 230: [: too many arguments
~/.oh-my-zsh/custom/plugins/git-trim/git-trim: line 230: [: too many arguments
~/.oh-my-zsh/custom/plugins/git-trim/git-trim: line 230: [: too many arguments
``` 

Line in question: 
```sh
if [ -z $(git log -1 -s --since="3 months ago" $branch) ]; then
```

**Comment by jasonmccreary** [comment]
Yeah, I think there's some variable expansion happening here. Are you using the latest script from `main`?

**Comment by mhdcodes** [comment]
@jasonmccreary yes, I'm using the latest commit from `main`

**Comment by jasonmccreary** [comment]
This was actually part of a larger bug. Should be fixed in 06f8d76.

**Comment by mhdcodes** [comment]
@jasonmccreary thanks working now.
